testing/test_decimal_approx.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+import decimal
+
+import pytest
+
+
+def test_decimal_approx_repr_issue():
```

Comment:
```suggestion
def test_decimal_approx_repr_issue() -> None:
```

Reply:
Please move this test over to https://github.com/pytest-dev/pytest/blob/main/testing/python/approx.py. :+1: